tools/QsLog/QsLog.cpp

```diff
@@ -25,6 +25,9 @@
 
 #include "QsLog.h"
 #include "QsLogDest.h"
+
+HIDE_COMPILE_WARNINGS
+

```

Comment:
Adding changes to QsLog files is kind of weird, but given how it's embedded in the project, I'm not sure that there's another way.